### DISCUSSION TITLE
How can I perform a code search without having to log in again and again?

### DISCUSSION THREAD

**Discussion by Michaelsy** [community/community#131011] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

If I want to carry out a code search, I have to log in again almost every time (using a 2FA). This is extremely inconvenient because I use an OTP app as 2FA and have to type in the OT password manually. (Which 2FA would perhaps be more convenient and could be used on both a tablet and a PC?)

The problem is that the login status is not saved or somehow maintained in the browser, as is usually the case. (With the exception of the banking area, of course).

BTW: I use Win7 and Android9 and therefore probably not the latest browsers. Could my problem have something to do with this?

**Comment by DNin01** [comment, marked as answer]
Android 9 and later support passkeys, and the GitHub Mobile app is one of the ways to authenticate with 2FA - you could try using either of those to speed up authentication.

If you aren't staying signed in after closing your browser, check that cookies are allowed. Maybe you blocked cookies or set them to clear when you close your browser, or you are using a private browsing window, which will clear cookies when closed.

**Reply by Michaelsy** [reply]
Many thanks for the reply! Oddly enough, the problem has been alleviated for me (I stay logged in longer), although I don't know how that came about. The only difference I'm aware of is that I now have an OTP app installed on the mobile device I want to stay logged into Github on. (Previously I had this app on a different device).  **Edit:** Perhaps I have also updated the browser (Android 9 / Chrome) I am using for Github.
